classic/src/view/form/Print.js

```diff
@@ -41,135 +59,32 @@ Ext.define("MoMo.client.view.form.Print", {
         this.provider = Ext.create('GeoExt.data.MapfishPrintProvider', {
             url: this.getUrl() + appCombo.getValue() + '/capabilities.json',
             listeners: {
-                ready: 'onPrintProviderReady',
+                ready: function(){
+                    this.getController().onPrintProviderReady(this.provider);
```

Comment:
You can access the view from the controller by executing ``this.getView()`` (where ``this`` is the controller instance), so ``this.getView().provider`` should do the trick or at least did it for me in former times.